docs/GettingStarted.asciidoc

```diff
@@ -194,6 +194,24 @@ tags.
 }
 -------------------------------------------------------------------
 
+==== Areas ====
+There are three kinds of areas:
+
+* *Regular areas* define relevant parts of the screenshot. Those must match
+  with at least the specified similarity percentage. Regular areas are
+  displayed as green boxes in the needle editor and as green or red frames
+  in the needle view.
+* *OCR areas* also define relevant parts of the screenshot. However, an OCR
+  algorithm is used for matching. In the needle editor exclude areas are
+  displayed as orange boxes. To turn a regular area into an OCR area within
+  the needle editor, double click the concerning area twice. Note that such
+  needles are only rarely used.
+* *Exclude areas* can be used within regular areas to ignore parts of it.
```

Comment:
"within regular areas to ignore parts of it." -> That's misleading as match areas only define the starting point and the reference picture but the whole screen can be searched for a match. That is why an exclude area can be used anywhere on the screen and it makes sense to do so.